business/istio_config.go

```diff
@@ -587,16 +587,15 @@ func (in *IstioConfigService) ParseJsonForCreate(resourceType, subresourceType s
 	} else {
 		kind = kubernetes.PluralType[resourceType]
 	}
+	var pivot interface{}
 	switch resourceType {
 	case Gateways:
 		istioConfigDetail.Gateway = &models.Gateway{}
 		err = json.Unmarshal(body, istioConfigDetail.Gateway)
```

Comment:
Note that Wizards also create Gateways, so same issue would happen there as well.

Reply:
I tested and seems to work OK. I guess it's because the wizards don't try to create `null` attributes.